canonical-web-and-design/ubuntu.com
# Redirect old vmware landing page

## Done

* Redirecting former vmware wbn engage page to newer one

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure /engage/vmware-to-openstack redirect to /engage/vmware-to-charmed-openstack
